packages/block-editor/src/components/rich-text/index.js

```diff
@@ -141,9 +150,17 @@ export function RichTextWrapper(
 			selectionStart: isSelected ? selectionStart.offset : undefined,
 			selectionEnd: isSelected ? selectionEnd.offset : undefined,
 			isSelected,
+			blockBindings,
+			getBlockBindingsSource,
```

Comment:
For this it's better to use `const { getBlockBindingsSource } = useSelect(...)`

Reply:
You mean outside the current `selector`, right? I can do that 🙂

Reply:
I've changed that as part of [this commit](https://github.com/WordPress/gutenberg/pull/58085/commits/aaf8652d0e1facf6f1cd7431a4f96819e1f59c1b).